packages/cactus-plugin-satp-hermes/src/main/typescript/core/recovery/rollback/rollback-strategy-factory.ts

```diff
@@ -0,0 +1,62 @@
+import { Logger, LoggerProvider } from "@hyperledger/cactus-common";
+import { SATPSession } from "../../satp-session";
+import { Stage0RollbackStrategy } from "./stage0-rollback-strategy";
+import { Stage1RollbackStrategy } from "./stage1-rollback-strategy";
+import { Stage2RollbackStrategy } from "./stage2-rollback-strategy";
+import { Stage3RollbackStrategy } from "./stage3-rollback-strategy";
+import { RollbackLogEntry } from "../../../generated/proto/cacti/satp/v02/crash_recovery_pb";
+import { SATPBridgeManager } from "../../stage-services/satp-bridge/satp-bridge-manager";
+import { SATPBridgeConfig } from "../../types";
+
+export interface RollbackState {
+  currentStage: string;
+  // todo add rollback state
```

Comment:
Please add the rollback state struct in the proto definition. Please import the Rollback state from the protos instead of using this interface.